### ISSUE TITLE
No error message is shown after a declined payment

### ISSUE THREAD

**Issue by ghost** [AltaPay/magento2#4]
During a checkout, when the payment is declined for some reason, the user is redirected to the shipping page, but no error message is shown to the user. You can reproduce this problem using the credit card number 4180000000000566 (it will be rejected).

**Comment by lsv** [comment]
Is the error messages in body response?
Header > ErrorMessage ?

**Comment by ghost** [comment]
Yes, the URL `<magento backend>/sdmaltapay/index/fail` will be called by the gateway.

The gateway will send, for example:

```
<?xml version="1.0"?>
<APIResponse version="20160719">
    <Header>
        <Date>2017-01-04T14:42:12+01:00</Date>
        <Path>API/reservationOfFixedAmount</Path>
        <ErrorCode>0</ErrorCode>
        <ErrorMessage/>
    </Header>
    <Body>
        <Result>Failed</Result>
        <MerchantErrorMessage>TestAcquirer[amount=5.66 case]</MerchantErrorMessage>
        <CardHolderErrorMessage>Card Declined</CardHolderErrorMessage>
        <CardHolderMessageMustBeShown>false</CardHolderMessageMustBeShown>
        <Transactions>
            <Transaction>
```

**Comment by lsv** [comment]
Which key do you want me to write in a message? just CardHolderErrorMessage ?

**Comment by ghost** [comment]
Yes, exactly.

**Comment by ghost** [comment]
The error message is still not being shown for the customer.

**Comment by lb-altapay-com** [comment]
This was test with Magento 2.1.7, 2.1.8 and 2.1.9 . The issue is still present.

**Comment by sz-altapay-com** [comment]
This issue has been fixed in version 0.1.9.
